Batch8CucumberSeleniumAPI/pom.xml

```diff
@@ -105,17 +105,6 @@
 			<version>3.10.2</version>
 		</dependency>
 
-		<dependency>
```

Comment:
Why these unnecessary changes in pom.xml file, please do make changes only where it needed

Reply:
In my mission whenever a i am adding log dependancies dont know why but  that causes the error, so i add log4j jar in  my project.